src/proxy/http/CMakeLists.txt

```diff
@@ -45,7 +45,7 @@ if(BUILD_REGRESSION_TESTING)
   target_sources(http PRIVATE RegressionHttpTransact.cc)
 endif()
 
-target_include_directories(http PRIVATE ${YAMLCPP_INCLUDE_DIR} OpenSSL::SSL)
+target_include_directories(http PRIVATE ${YAMLCPP_INCLUDE_DIR})
```

Comment:
This line can be removed as well.  http doesn't include yaml-cpp and if it did then it should be listed as a dependency below.

Reply:
done